content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -31,10 +27,13 @@ With a waypoint proxy deployed, you can use the following traffic route types:
 
 |  Name  | Feature Status | Attachment |
 | --- | --- | --- |
-| [`HTTPRoute`](https://gateway-api.sigs.k8s.io/guides/http-routing/) | Beta | `parentRefs` |
+| [`HTTPRoute`](https://gateway-api.sigs.k8s.io/guides/http-routing/) | Stable | `parentRefs` |
+| [`GRPCRoute`](https://gateway-api.sigs.k8s.io/guides/grpc-routing/) | Stable | `parentRefs` |
 | [`TLSRoute`](https://gateway-api.sigs.k8s.io/guides/tls) | Alpha | `parentRefs` |
 | [`TCPRoute`](https://gateway-api.sigs.k8s.io/guides/tcp/) | Alpha | `parentRefs` |
 
+(TLS and TCP routing are stable features in Istio, but support for these objects remains at Alpha because the Gateway API objects are still in the experimental channel.)
```

Comment:
That's tough, so there is no stable way of managing tcp traffic in ambient.

Reply:
Yep - and it's really because Gateway API is (rightly) hesitant about pushing those APIs forward without implementations that are championing those use-case.